flooder/Cargo.toml

```diff
@@ -1,17 +1,18 @@
 [package]
 name = "flooder"
 version = "0.1.0"
+authors = ["Cardinal Cryptography"]
```

Comment:
in other packages we do not admit so; not a blocker, but I think we should be consistent